taskcat/deployer.py

```diff
@@ -407,22 +425,25 @@ def _get_file_list(self, input_path):
             self._file_list = _file_list
         return self._file_list
 
-    def _string_rewriter(self, current_string, replacement_bucket_name):
+    def _string_rewriter(self, current_string):
         if self._prod_bucket_name in current_string:
             # If the path is s3/http/https
             if any(x in current_string for x in ['s3:', 'http:', 'https:']):
                 # Make sure that it's part of the target key prefix (that is, part of this repo)
                 if self._target_key_prefix in current_string:
```

Comment:
This check is interesting and needs a second look now that we have source key prefix in play. This was added to help templates that hardcode URLs to paths outside of the repo itself. So if it's pulling something from a remote location (like an installer or script), we would have to relocate it to a new place if it's part of the repo.... otherwise we leave it as is... since we can't control remote locations.